server/views/pages/home.njk

```diff
@@ -19,28 +19,43 @@
     </div>
     <div class="homepage-content">
         <div class="govuk-width-container">
-            <ul class="govuk-grid-row card-group">
-                <li class="govuk-grid-column-one-third card-group__item">
-                    {{ card({
-                        "href": '/confirm-arrival/choose-prisoner',
-                        "clickable": "true",
-                        "heading": 'People booked to arrive today',
-                        "description": 'Confirm the arrival of prisoners booked to arrive today.',
-                        "data-qa": "choose-prisoner"
-                    }) }}
-                </li>
-                {% if temporaryAbsencesEnabled %}
+            <h2 class="govuk-heading-m govuk-!-margin-top-7">Prepare for and confirm new arrivals</h2>
+                <ul class="govuk-grid-row card-group">
                     <li class="govuk-grid-column-one-third card-group__item">
                         {{ card({
-                            "href": '/prisoners-returning',
+                            "href": '/confirm-arrival/choose-prisoner',
                             "clickable": "true",
-                            "heading": 'People returning from temporary absence',
-                            "description": 'Confirm the return of prisoners out on temporary absence.',
-                            "data-qa": "return-from-temporary-absence"
+                            "heading": 'People booked to arrive today',
+                            "description": 'Confirm the arrival of prisoners arriving from court, a custody suite or another establishment.',
+                            "data-qa": "choose-prisoner"
                         }) }}
                     </li>
-                {% endif %}
-            </ul>
+                    {% if temporaryAbsencesEnabled %}
```

Comment:
TEMPORARY_ABSENCE_ENABLED is true across dev, pre and prod so can remove this conditional check across the app.